projects/strkfarm/erc4626.js

```diff
@@ -0,0 +1,48 @@
+const ERC4626Abi = [
+  {
+    "name": "asset",
+    "type": "function",
+    "inputs": [],
+    "outputs": [
+      {
+        "type": "core::starknet::contract_address::ContractAddress"
+      }
+    ],
+    "state_mutability": "view"
+  },
+  {
+    "name": "balanceOf",
+    "type": "function",
+    "inputs": [
+      {
+        "name": "account",
+        "type": "core::starknet::contract_address::ContractAddress"
+      }
+    ],
+    "outputs": [
+      {
+        "type": "core::integer::u256"
+      }
+    ],
+    "state_mutability": "view",
+    "customInput": 'address',
```

Comment:
@akiraonstarknet so, this is needed for some reason, dont ask me why (I did some hack long time ago), your code was failing because of this line